VirtualMachineSetup.md

```diff
@@ -0,0 +1,152 @@
+
+- **Hardware/CPU & Memory:** Change Memory to at least 8GB. 16GB or more is preferred if your computer has enough RAM. 2 processors should be fine, but if you feel the need you can increase that as well.
+ 
+ - **Hardware/Shared Printers:** Uncheck *Share Mac printers with Debian GNU/Linux*.
+
+ - **Hardware/Sound & Camera:** Uncheck *Share Mac camera with Linux*
+
+ - **Backup** Check the box for *SmartGuard* in order to enable automatic snapshots of your VM. This is highly recommended as long as you have the storage space. Note that you can also manually take snapshots at any time. Click *Details...* to customize how often snapshots are taken and how many are kept on your machine. If you aren't sure just keep the default settings.
+
+
+Other settings that may not be present in newer versions of Parallels (in which case you can safely ignore)
+
+- **Options/Applications:**  un-check Share Linux applications with Mac unless you enjoy clutter or have a specific need.
+- **Options/More Options:** check Authenticate with macOS SSH public key.
+
+When you are done, close the Configuration modal, and click *Continue*
+
+## Installing Debian 
+
+Once you boot your new virtual machine in Debian you will be prompted with the install guide. 
+
+1. Chose either *Install* or *Graphical Install*, the latter will be used in this guide but it is just a slightly nicer UI, all of the steps and options are the same. 
+
+![Debian Install Screen](setup_screenshots/install_screen.png?raw=true)
+
+2. Select your language, location, and keyboard layout on the next few screens. Debian will then begin installing. 
+
+3. *Configure the network* You will be asked for a Hostname. Since we aren't running a network this doesn't really matter, you can leave the default `debian`.
+
+![Debian Install Screen](setup_screenshots/hostname.png?raw=true)
+
+You will then be asked for a domain name, this can be left blank.
+
+![Debian Install Screen](setup_screenshots/domain_name.png?raw=true)
+
+4. *Set up users and passwords* When prompted chose a password for your root user. Debian will create both a `root` user and another user account for you. This security measure is a bit of overkill for us since we are just developing on this VM, so it may simplify things to use the same password for both. 
+
+Next you will be prompted to set up your non-root user account. You will be asked for the real name of the user, a username, and a password. It will make things slightly simpler to use the same username as your home computers user account but you can chose anything you want here. 
+
+Chose your timezone and click continue to continue the installation. 
+
+6. *Partition Disks* You will be prompted on how to partition the disks. The production machines use LVM but for development it should be fine to just chose *Guided - use entire disk*. 
```

Comment:
@umbernhard Review my advice here.

Reply:
that's right, I'd be interested to hear other's thoughts on whether we should use the same partitioning on dev as we do on prod. @benadida @eventualbuddha thoughts? It's not that much harder to set up.

Reply:
I don't think this will matter all that much, especially since the file path layout *is* different in prof and dev and i don't anticipate they will become the same in dev and locked down... So in interest of simplicity I'd say dev could stick to just one partition and be done with it.

I'm open to counter arguments though!

Reply:
I don't have a strong feeling on the matter.